mne/io/ctf/ctf.py

```diff
@@ -105,10 +105,17 @@ def __init__(self, directory, system_clock='truncate', preload=False,
         res4 = _read_res4(directory)  # Read the magical res4 file
         coils = _read_hc(directory)  # Read the coil locations
         eeg = _read_eeg(directory)  # Read the EEG electrode loc info
+
         # Investigate the coil location data to get the coordinate trans
         coord_trans = _make_ctf_coord_trans_set(res4, coils)
+
+        digs = _read_pos(directory, coord_trans)
+
         # Compose a structure which makes fiff writing a piece of cake
         info = _compose_meas_info(res4, coils, coord_trans, eeg)
+        if len(digs):
```

Comment:
This should actually be ``len(digs) > 0``, although it's basically doing the same thing. I'll change it.